### PR TITLE
[Automation] Add GH action to add issues to ingest board

### PR DESCRIPTION
**Pull request by kpollich**
Issues labeled w/ the control plane or data plane labels in this repository will be added to the ingest project w/ `Area: Elastic Agent` set

### CODE REVIEW

**Review by cmacknz — APPROVED** [review]
There are a bunch of repositories we will need to port this to, Beats and all of the elastic-agent-* repositories. CC @pierrehilbert

File: .github/workflows/add-issues-to-ingest-board.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+              addProjectV2ItemById(input: { projectId: $project_id, contentId: $content_id }) {
+                  item {
+                    id
+                  }
+                }
+              }
+          project_id: ${{ env.INGEST_PROJECT_ID }}
+          content_id: ${{ github.event.issue.node_id }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.FLEET_TECH_KIBANA_USER_TOKEN }}
```
[the remark this answers is not in the document]
Comment (cmacknz) [reply .github/workflows/add-issues-to-ingest-board.yml]: I just granted you admin access to the repo here so you can set this up.

[the remark this answers is not in the document]
Comment (kpollich) [reply .github/workflows/add-issues-to-ingest-board.yml]: Thanks. I added the proper PAT as a `PROJECT_ASSIGNER_TOKEN` repository secret so this should be good to go.

**Comment by kpollich** [comment]
> There are a bunch of repositories we will need to port this to, Beats and all of the elastic-agent-* repositories.

I'll go through all of these and copy/paste. Thanks for flagging.

**Comment by kpollich** [comment]
@elasticmachine merge upstream

**Pull request merged**